docs/common/module/coffee-module-mp-opentracing.adoc

```diff
@@ -0,0 +1,70 @@
+[#common_module_coffee-module-mp-restclient]
+= coffee-module-mp-opentracing
+
+A modul célja a microprofile opentracing támogatása mely tartalmazza a következő alapelveket:
+
+* Coffee kompatibilizás - lehetőséget biztosít hogy a coffee moduljai trace információkat szolgáltassanak.
+* Metrika és logolást egészíti ki a teljes observability érdekében.
+* Jelenleg saját osztályokat használunk mivel nem lehet extension szinten jól kezelni azt hogy melyik komponens adjon trace információkat.
+
+== microprofile-opentracing
+Alapja a https://github.com/eclipse/microprofile-opentracing valamint a
+https://github.com/opentracing-contrib/java-interceptors projekt.
+Az `org.eclipse.microprofile.opentracing.Traced` annotáció helyett saját bindingot használunk, mivel jelenleg a beépített interceptor
+nem tudja felolvasni a dinamikusan hozzáadott InterceptorBindingot.
+
+== Core
+Az extension azokat a classokat amik `@Traceable` annotációval vannak jelölve, megkapják a `hu.icellmobilsoft.coffee.module.mp.opentracing.extension.Traced` bindingot.
+Ezt használja fel az `OpenTraceInterceptor`. Ezzel a megoldással később könnyen lehet cserélni/módosítani a trace interceptor működését.
+
+=== coffee-module-etcd trace
```

Comment:
Más modulra nem kell/nem akarjuk?

Reply:
A redist próbáltam, de nagyon bele kellene nyulni a repository részébe hogy ez működjön, én most csak az etcd-n hagynám egy darabig pilotban.